### ISSUE TITLE
Fix py3k syntax issues in twisted/python/usage.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7157]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7157|
|Type|enhancement|
|Created|2014-04-15 19:23:09Z|
|Branch|https://github.com/twisted/twisted/tree/usage.py-py3-7157|

<details><summary>Searchable metadata</summary>

```
trac-id__7157 7157
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_usage_py_py3_7157 branches/usage.py-py3-7157
branch_author__radix radix
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1397589789000000 1397589789000000
changetime__1474198049187875 1474198049187875
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is solved by https://github.com/twisted/twisted/pull/14

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix commented|
|-|-|

(In [42274]) Branching to usage.py-py3-7157.

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_3"></a>|@radix set status to `closed`|
|-|-|

(In [42280]) Merge usage.py-py3-7157: usage.py in python3 syntax

Author: Alex
Reviewer: radix
Fixes: [#7157](https://github.com/twisted/twisted/issues/7157)

Update twisted/python/usage.py to have Python 3-compatible syntax.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
